modules/pool/pool.gql

```diff
@@ -1503,37 +1503,8 @@ interface GqlPoolEvent {
     poolId: String!
     timestamp: Int!
     valueUSD: Float!
-}
-
-type GqlPoolAddRemoveEventV3 implements GqlPoolEvent {
-    chain: GqlChain!
-    id: ID!
-    tx: String!
-    logIndex: Int!
-    blockNumber: Int!
-    blockTimestamp: Int!
-    type: GqlPoolEventType!
-    sender: String!
-    userAddress: String!
-    poolId: String!
-    timestamp: Int!
-    valueUSD: Float!
-    tokens: [GqlPoolEventAmount!]!
-}
-
-type GqlPoolSwapEventV3 implements GqlPoolEvent {
-    chain: GqlChain!
-    id: ID!
-    tx: String!
-    logIndex: Int!
-    blockNumber: Int!
-    blockTimestamp: Int!
-    type: GqlPoolEventType!
-    sender: String!
-    userAddress: String!
-    poolId: String!
-    timestamp: Int!
-    valueUSD: Float!
-    tokenIn: GqlPoolEventAmount!
-    tokenOut: GqlPoolEventAmount!
+    tokens: [GqlPoolEventAmount!]
+    tokenIn: GqlPoolEventAmount
+    tokenOut: GqlPoolEventAmount
```

Comment:
is `tokens` for add/remove and `tokenIn`/`tokenOut` for swap?

Reply:
yes, like you said